### PR TITLE
[Containerapp] 'az containerapp create/update': Support --max-inactive-revisions

### PR DESCRIPTION
**Pull request by njuCZ**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 1]
```diff
@@ -2350,3 +2350,21 @@ def test_containerapp_termination_grace_period_seconds(self, resource_group):
...
+    @ResourceGroupPreparer(location="northeurope")
+    def test_containerapp_max_inactive_revisions(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        app = self.create_random_name(prefix='aca', length=24)
+        image = "mcr.microsoft.com/k8se/quickstart:latest"
+        maxInactiveRevisions = 100
+        env = prepare_containerapp_env_for_app_e2e_tests(self)
+
+        self.cmd(f'containerapp create -g {resource_group} -n {app} --image {image} --ingress external --target-port 80 --environment {env} --max-inactive-revisions {maxInactiveRevisions}')
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:2364]: Please test command create/update without `--max-inactive-revisions`

Comment (njuCZ) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:2368]: updated

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 2]
```diff
@@ -2350,3 +2350,26 @@ def test_containerapp_termination_grace_period_seconds(self, resource_group):
...
+    def test_containerapp_max_inactive_revisions(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        app = self.create_random_name(prefix='aca', length=24)
+        image = "mcr.microsoft.com/k8se/quickstart:latest"
+        
+        env = prepare_containerapp_env_for_app_e2e_tests(self)
+
+        self.cmd(f'containerapp create -g {resource_group} -n {app} --image {image} --ingress external --target-port 80 --environment {env} --cpu 0.5 --memory 1Gi')
+        self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[JMESPathCheck("properties.configuration.maxInactiveRevisions", None)])
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:2365]: Please add test for create/update `properties.configuration.maxInactiveRevisions` with --yaml

**Review by Greedygre — APPROVED** [review]

**Comment by Greedygre** [comment]
Recommend title:

[Containerapp] 'az containerapp create/update': Support --max-inactive-revisions

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**